lib/simp/rake/build/pkg.rb

```diff
@@ -46,7 +46,9 @@ def define_tasks
           @build_dirs = {
             :modules => get_module_dirs(args[:method]),
             :aux => [
+              # Legacy rsync, keep it for backwards-compatibility
               "#{@src_dir}/rsync",
+              "#{@src_dir}/rsync_skeleton",
```

Comment:
This shouldn't be called 'rsync_skeleton' on the filesystem.

The repo can be 'rsync_skeleton' but the directory should just be 'rsync', just like the 'environment_skeleton'